data/flashcards.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "id": "1",
+    "englishWord": "apple",
+    "japaneseWord": "りんご",
+    "interval": 1,
+    "repetition": 0,
+    "easiness": 2.5,
+    "nextReview": "2024-09-26T00:00:00.000Z",
+    "audioSource": "../assets/audio/apple.mp3"
+  },
+  {
+    "id": "2",
+    "englishWord": "dog",
+    "japaneseWord": "犬",
+    "interval": 1,
```

Comment:
What do these signify? 

I'm guessing 

- easiness always starts at 2.5 and gets adjusted based on how hard it is? Maybe it's like 'difficulty' ?
- repetition is just the number of times it's been reviewed
- interval ??

Reply:
Yeah that's the general idea, with interval being the length of time between the previous and next appearance. 

From the creator of the algorithm:

```
Formula to calculate inter-repetition intervals:

I(1):=1
I(2):=6
for n>2 I(n):=I(n-1)*EF

where:

I(n) - inter-repetition interval after the n-th repetition (in days)
EF - easiness factor reflecting the easiness of memorizing and retaining a given item in memory (later called the E-Factor).
```

[https://super-memory.com/english/ol/sm2.htm](url)

Reply:
cool thanks for laying that out. Makes sense.